src/AbstractPhpdocToTypeDeclarationFixer.php

```diff
@@ -43,6 +43,7 @@ abstract class AbstractPhpdocToTypeDeclarationFixer extends AbstractFixer implem
         'iterable' => 7_01_00,
         'object' => 7_02_00,
         'mixed' => 8_00_00,
+        'union-types' => 8_00_00,
```

Comment:
I would prefer to put the `8_00_00` is code directly, dash cannot be present in type, thus it is safe, but overall it is only confusing.